index.html

```diff
@@ -25,6 +25,7 @@
 				{title: 'Vorlesung 5', path: '/vorlesung5/vorlesung5'},
 				{title: 'Vorlesung 6', path: '/vorlesung6/vorlesung6'},
 				{title: 'Vorlesung 7', path: '/vorlesung7/vorlesung7'}
```

Comment:
Hier fehlt das Komma, in dieser form würde Docute sich weigern die Webseite aufzubauen